scripts/geo.py

```diff
@@ -12,171 +12,75 @@
-
-                optional arguments:
-                -h, --help            show this help message and exit
-                -l LOC, --loc LOC     Location name
-                -g, --graph           Graph station types [False]
-                -v {l,m,h}, --verbosity {l,m,h}
-                                    Verbosity level: 'l', 'm', or 'h'. [h].
-                --date DATE           UTC YYYY/MM/DD or '<' or '>' or 'n/a' or 'now' [now]
-                --time TIME           UTC hh:mm or float (hours)
-                -x {N,n,E,e,Z,z}, --xgraph {N,n,E,e,Z,z}
-                                    X-axis of graph. [E]
-                -y {N,n,E,e,Z,z}, --ygraph {N,n,E,e,Z,z}
-                                    Y-axis of graph. [N]
-                -t STATION_TYPES, --station-types STATION_TYPES
-                                    Station types used for input (csv_list or all) [HH]
-                --show-state {active,all}
-                                    Show only the 'active' stations or 'all' ['all']
-                --show-label {name,num,ser}
-                                    Label by station_name (name), ant_num (num) or
-                                    serial_num (ser) or false [num]
-                --fig-num FIG_NUM     Provide a specific figure number to the plot [default]
-                --update UPDATE       Update station records. Format station0:col0:val0,
-                                    [station1:]col1:val1... [None]
-                --add-new-geo         Flag to enable adding of a new geo_location under
-                                    update. [False]
-        """
-        )
-        sys.exit()
-
     # interpret args
     at_date = cm_utils.get_astropytime(args.date, args.time)
     args.position = cm_utils.listify(args.position)
     args.station_types = cm_utils.listify(args.station_types)
     args.show_label = args.show_label.lower()
-    if args.show_label == 'false':
+    if args.show_label.lower() == 'false':
         args.show_label = False
-    if args.fig_num.lower() == 'default':
-        args.fig_num = args.xgraph + args.ygraph
     show_fig = False
-
-    # package up state to dictionary
-    state_args = {'verbosity': args.verbosity.lower(),
-                  'xgraph': args.xgraph.upper(),
-                  'ygraph': args.ygraph.upper(),
-                  'station_types': args.station_types,
-                  'show_state': args.show_state.lower(),
-                  'show_label': args.show_label,
-                  'fig_num': args.fig_num}
+    xgraph = args.xgraph.upper()
+    ygraph = args.ygraph.upper()
+    show_state = args.show_state.lower()
+    if args.action == 'sin':
+        cutoff = at_date
+        at_date = cm_utils.get_astropytime('now')
 
     # start session and instances
     db = mc.connect_to_mc_db(args)
     session = db.sessionmaker()
-    h = geo_handling.Handling(session)
+    G = geo_handling.Handling(session)
```

Comment:
Single-letter local variable names should not be capitalized.

Reply:
Small push-back -- when setting up a local class using cap is allowed (well, obviously that is not said anywhere, except in this comment but...)

Reply:
It's one of those things that just Is Not Done in Python ([cf PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)).